dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/handler/HandlerUtilTest.java

```diff
@@ -0,0 +1,156 @@
+package com.dotcms.enterprise.publishing.remote.handler;
+
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.MultiTree;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class HandlerUtilTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void setMultiTree_on_diff_orders_should_not_reorder() throws IOException, DotSecurityException, DotDataException {
```

Comment:
I would add a comment on this test to understand with more details the scenario being tested and the expected result